internal/.env

```diff
@@ -0,0 +1,2 @@
+OPENAI_API_KEY=XX
+GPT_SYSTEM_PROMPT=You are a helpful robot
```

Comment:
Again, please don't write the system prompts, that's my job (and "You are a helpful robot" will not produce expected results, it needs to be a lot more direct and complex). Replace it with a placeholder for now (could literally be "PLACEHOLDER").